pages/blog.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import Blog from '../components/Blog'
+
```

Comment:
The blog page needs to follow other page conventions and import and use the `head` element. See the [mission page source code](https://github.com/teacherfund/TeacherFund_next/blob/master/pages/mission.js) for an example. Right now